### PR TITLE
feat: freeze project update freeze in task project

### PR DESCRIPTION
**Pull request by vishdha**
[ASANA](https://app.asana.com/0/1192407894480744/1200683289673708)

Peek 2021-08-04 15-42

### CODE REVIEW

File: erpnext/public/js/utils.js [hunk 1]
```diff
@@ -840,4 +840,60 @@ $(document).on('app_ready', function() {
 			}
 		});
 	});
+
+	$.each(["Project", "Task"], function (i, doctype) {
```
Comment (hrwX) [inline erpnext/public/js/utils.js:844]: - move this in project.js

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/public/js/utils.js [hunk 2]
```diff
@@ -840,4 +840,60 @@ $(document).on('app_ready', function() {
...
+									.filter((link) => link.doctype === doctype)
+									.map((link) => frappe.utils.get_form_link(link.doctype, link.name, true))
+									.join(", ");
+								links_text += `<li><strong>${doctype}</strong>: ${docnames}</li>`;
+
+							}
+							links_text = "<ul>" + links_text + "</ul>";
+
+							confirm_message = `${frm.doc.name.bold()} is linked with the following documents: ${links_text}`;
+							frappe.confirm(__(`${confirm_message} Do you want to set them as <strong>{0}</strong> too?`, [frm.doc.freeze ? "Freeze" : "Not Freeze"]),
```
Comment (hrwX) [inline erpnext/public/js/utils.js:875]:
```suggestion
							frappe.confirm(__(`{0} Do you want to set them as <strong>{1}</strong> too?`, [confirm_message, frm.doc.freeze ? "Freeze" : "Not Freeze"]),
```

File: erpnext/public/js/utils.js [hunk 3]
```diff
@@ -840,4 +840,60 @@ $(document).on('app_ready', function() {
...
+								let docnames = links
+									.filter((link) => link.doctype === doctype)
+									.map((link) => frappe.utils.get_form_link(link.doctype, link.name, true))
+									.join(", ");
+								links_text += `<li><strong>${doctype}</strong>: ${docnames}</li>`;
+
+							}
+							links_text = "<ul>" + links_text + "</ul>";
+
+							confirm_message = `${frm.doc.name.bold()} is linked with the following documents: ${links_text}`;
```
Comment (hrwX) [inline erpnext/public/js/utils.js:874]:
```suggestion
							confirm_message = __(`{0} is linked with the following documents: {1}`, [frm.doc.name.bold(), links_text]);
```

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by vishdha** [comment]
Closing this. Requested changes added in new PR
https://github.com/Bloomstack/erpnext/pull/1493

**Pull request closed without merging**